auth/app/auth/AuthController.scala

```diff
@@ -36,14 +39,23 @@ class AuthController(auth: Authentication, config: AuthConfig,
     val firstName = user.firstName
     val lastName = user.lastName
 
+    val showPaidFuture: Future[Boolean] = hasPermission(PandaUser(request.user), Permissions.ShowPaid)
+
+
+    val showPaid: Boolean = Await.result(showPaidFuture, 100.millis)
```

Comment:
I'm pretty sure this is gross and horrible but I'm not sure how else I can do it, and the result _does_ rely on it after all

Reply:
Use `auth.AuthAction.async` in `def index` below and then restructure the index response to map over the future:

```scala
hasPermission(PandaUser(request.user), Permissions.ShowPaid).map { canSeePaid =>
   respond(
      ... blah blah ...
   )
}
```

tbh we could just copy the whole lot into the method, no need for it to be val